rts/Sim/Units/UnitTypes/Builder.cpp

```diff
@@ -729,6 +730,18 @@ bool CBuilder::StartBuild(BuildInfo& buildInfo, CFeature*& feature, bool& inWait
 
 	buildInfo.pos = CGameHelper::Pos2BuildPos(buildInfo, true);
 
+	auto isBuildeeFloating = [](BuildInfo& buildInfo) {
```

Comment:
perhaps this could be `bool BuildInfo::isBuildeeFloating() const`? but it's weird that floating matters, see below

Reply:
Done.